### PR TITLE
LPD-37881 Create and manage product configurations

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://liferay.atlassian.net/browse/LPD-37881

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-service/service.xml [hunk 1]
```diff
@@ -303,6 +303,134 @@
...
+		<column name="userName" type="String" />
+		<column name="createDate" type="Date" />
+		<column name="modifiedDate" type="Date" />
+
+		<!-- Other fields -->
+
+		<column name="CPConfigurationListId" type="long" />
+		<column name="classNameId" type="long" />
+		<column name="classPK" type="long" />
+		<column name="CPTaxCategoryId" type="long" />
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-service/service.xml]: sort

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Review by alessiorendina — APPROVED** [review]
LGTM

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**